docs/services/inference-endpoints.md

```diff
@@ -378,6 +378,7 @@ Models are organized by cluster and marked with the following capabilities:
 ??? "Chat Language Models"
 
     - gpt-oss-120b-131072<sup>H</sup>
+    - Llama-4-Maverick-17B-128E-Instruct<sup>H</sup>
```

Comment:
https://squidfunk.github.io/mkdocs-material/reference/formatting/#sub-and-superscripts

```suggestion
    - Llama-4-Maverick-17B-128E-Instruct^H
```